packages/core/src/loaders/http/httpLoader.ts

```diff
@@ -0,0 +1,26 @@
+import { FilesystemNodeType } from '@stoplight/graphite/backends/filesystem';
+import { IHttpOperation } from '@stoplight/types';
+import axios from 'axios';
+import trimStart = require('lodash/trimStart');
+import { extname } from 'path';
+import { IHttpLoaderOpts } from '../../types';
+import { GraphFacade } from '../../utils/graphFacade';
+
+export class HttpLoader {
+  constructor(private graphFacade: GraphFacade) {}
+
+  public async load(opts?: IHttpLoaderOpts): Promise<IHttpOperation[]> {
+    if (!opts || !opts.url) return [];
```

Comment:
What is the benefit or loading this method with an empty array?

Reply:
It's to be compliant with filesystemLoader. Filesystem loader returns empty array if spec is not set.

The other question is if it makes sense? For me not. But I don't think this PR is a good place for changing that. Changing that requires minor interface modifications.